spring-cloud-gcp-data-spanner/src/test/java/com/google/cloud/spring/data/spanner/core/mapping/event/AfterExecuteDmlEventTest.java

```diff
@@ -46,8 +46,8 @@ public void equalsHashcodeTest() {
 		assertThat(afterExecuteDmlEventa1).isNotEqualTo(null);
 		assertThat(afterExecuteDmlEventa1).isNotEqualTo(new Object());
 
-		assertThat(afterExecuteDmlEventa1.hashCode()).isEqualTo(afterExecuteDmlEventa1.hashCode());
-		assertThat(afterExecuteDmlEventa1.hashCode()).isEqualTo(afterExecuteDmlEventa1x.hashCode());
+		assertThat(afterExecuteDmlEventa1).hasSameHashCodeAs(afterExecuteDmlEventa1.hashCode());
+		assertThat(afterExecuteDmlEventa1).hasSameHashCodeAs(afterExecuteDmlEventa1x.hashCode());
```

Comment:
Does this pass? I'd think that the argument's `.hashCode()` needs to be removed as well.

Reply:
I suppose it passes because a hashcode of `Integer` is itself in primitive form...

Reply:
Fixed.